### PR TITLE
bugfix: add back "django.contrib.staticfiles", other adjusts

### PR DESCRIPTION
**Pull request by Eshaan7**

### CODE REVIEW

File: intel_owl/settings.py [hunk 1]
```diff
@@ -92,6 +92,7 @@ def retrieve(self, file, analyzer):
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
+    "django.contrib.staticfiles",
```
Comment (mlodic) [inline intel_owl/settings.py:95]: @Eshaan7 is this really necessary? I think Simone deleted it because those static files are just used in the readme

Comment (Eshaan7) [reply intel_owl/settings.py:95]: The django admin breaks without this

Comment (Eshaan7) [reply intel_owl/settings.py:95]: Because it needs css/JS static files to work

Comment (mlodic) [reply intel_owl/settings.py:95]: ah yeah right thanks to the explanation

**Pull request merged**